src/development/going-live.md

```diff
@@ -1,42 +1,94 @@
-3. If your domain is currently active elsewhere, the Time-To-Live (TTL) on your domain is set to the lowest possible value in order to minimize transition time.
-4. You have the auto-generated domain for your master branch.  This is the domain you see in the Location bar after selecting "Access site" in the UI.  You can also retrieve this value from the command line by running `platform environment:url` to see a list of all URLs that Platform.sh will serve for the current environment.  Write this down.
-5. Optional: If you want to guarantee that you have access to  your master environment before the domain name has switched over, use `ping` or any similar tool to determine the IP address of the master environment.  The IP address is not guaranteed stable but is unlikely to change during the course of the go-live process.
-6. Optional: If you want to use a 3rd party SSL certificate to encrypt your production site, you can obtain one from any number of 3rd party SSL issuers.  Platform.sh automatically provides SSL certificates for all sites issued by [Let's Encrypt](https://letsencrypt.org/) at no charge, and there is no charge for using a 3rd party SSL cert instead.
+<!-- toc -->
 
-## Configure your DNS provider
+## Pre-Launch Checklist.
 
-Configure your DNS provider to point your domain to your Platform.sh Master environment.
+Before you  go live you should go through the following check-list.
 
-The way to do so will vary somewhat depending on your registrar, but nearly all registrars should allow you to set a CNAME.  Some will call it an Alias or similar alternate name, but either way the intent is to say "this domain should always resolve to... this other domain".  Add a CNAME record from your desired domain (`www.example.com`) to the master environment hostname you wrote down earlier.
+### 1. Register a domain name with a supported provider
+You have a domain name registered for your site with a Registrar of your choice. The registrar must allow you to use CNAMEs for your domain.  (Some registrars may call these Aliases or similar.). If your domain is currently active elsewhere, the Time-To-Live (TTL) on your domain is set to the lowest possible value in order to minimize transition time.
+
+### 2. Note the target address (CNAME) for your DNS configuration
+ You have the auto-generated domain for your master branch.  This is the domain you see in "Access site" in the UI.
+
+ ![Master Access](/images/master-access.png)
+
+You can also retrieve this value from the command line by running `platform environment:url` to see a list of all URLs that Platform.sh will serve for the current environment.  
+ 
+ ![Platform Cli Master Url](/images/platform-cli-master-url.png)
+ 
+Write this down. This is what you will be using as the target for the CNAME you will configure with your DNS provider. Note that what you will see there is the URL, so you will need to remove http:// or https:// to get the name of the environment. 
```

Comment:
From my experience providing support, this is the part that causes the most disorientation for our users. We don't want a crucial piece of information to disappear from the screen just because they've assigned a domain.